consumer/nf_discovery.go

```diff
@@ -10,29 +10,69 @@ import (
 	"fmt"
 	"net/http"
 
+	ausf_context "github.com/omec-project/ausf/context"
```

Comment:
Follow the same naming convention already used in this file
```suggestion
	ausfContext "github.com/omec-project/ausf/context"
```